README.md

```diff
@@ -79,12 +124,31 @@ Bellow is a list of the supported API endpoints within the SDK.
 * Voucher Series
 * Vouchers
 * Way of Deliveries
- 
+
+### WebSockets
```

Comment:
Websockets or webhooks? @brokenprogrammer @BaconFinger

Reply:
The documentation we recieved says WebSocket API

Reply:
yeah, lets keep it like that :+1: